doc/quality.rst

```diff
@@ -457,6 +552,53 @@ H.265/HEVC
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-b:v $bitrate -minrate $bitrate -maxrate $bitrate`` | n2.8           | This triggers CBR.                                                       |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-bufsize $((2 * $bitrate))``                        | n4.0           | You can vary bufsize per your needs. We recommend to avoid going below 1 |
+|                                                       |                | second to avoid quality loss. Buffer size of 2 seconds is recommended    |
+|                                                       |                | for VBR.                                                                 |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-rc_init_occupancy $bitrate``                       | n2.8           | This is the initial buffer delay. You can vary this per your needs.      |
+|                                                       |                | Recommendation is to use 1/2 of bufsize.                                 |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| CBR & VBR common settings                                                                                                                         |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-extbrc 1 -look_ahead_depth 40``                    | n5.0           | This enables EncTools Software BRC (need to have look ahead depth > than |
+|                                                       |                | miniGOP size which is equal to bf+1).                                    |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-b_strategy 1 -bf 7``                               | master         | These 2 settings activate full 3 level B-Pyramid.                        |
++-------------------------------------------------------+----------------+--------------------------------------------------------------------------+
+| ``-refs 4``                                           | n2.8           | EncTools use Adaptive Long Term Reference (ALTR) feature which has lower |
+|                                                       |                | requirement for number of reference frames for HEVC encoding.            |
```

Comment:
Actually HEVC EncTools LTR is not supported.  Maybe best to say here "4 reference are recommended for high quality HEVC encoding."

Reply:
ok